docs/content/en/docs/implementing/tasks-non-k8s-apps.md

```diff
@@ -2,23 +2,46 @@
 title: Keptn for Non-Kubernetes Applications
 description: Using Keptn with Non-Kubernetes Applications
 weight: 95
-hidechildren: false # this flag hides all sub-pages in the sidebar-multicard.html
 ---
 
-It is possible to trigger Keptn Tasks for workloads and applications that are deployed outside of Kubernetes.
+Keptn Tasks can be triggered for workloads and applications
+that are deployed outside of Kubernetes.
+For example, Keptn could trigger load and performance tests
+for an application that is deployed on a virtual machine.
 
-For example, to trigger a load test for an application deployed on a virtual machine.
+To do this:
 
-You will still need to deploy Keptn on a Kubernetes cluster, but this can be a very lightweight, single-node kind cluster.
-Keptn's only job is to trigger on-demand tasks, so resource utilization will be minimal.
+1. [Install and enable a Kubernetes cluster](#install-and-enable-a-kubernetes-cluster)
+1. [Create a KeptnTaskDefinition](#create-a-keptntaskdefinition)
+1. [Create and apply a KeptnTask](#create-and-apply-a-keptntask)
+1. [Re-run the KeptnTask](#re-run-the-keptntask)
 
-## Step 1: Create a KeptnTaskDefinition
+## Install and enable a Kubernetes cluster
 
-When you have Keptn installed, [create a KeptnTaskDefinition](../implementing/tasks/).
+You must still
+[install](../install/install.md/#use-helm-chart)
+and
+[enable](../install/install.md/#enable-keptn-for-your-cluster)
+Keptn on a Kubernetes cluster,
+but this can be a very lightweight, single-node KinD cluster; see
```

Comment:
I really don't think we need instructions on how to create / run a kubernetes cluster. If a potentially user does not / cannot have a running cluster, they should first start learning what kubernetes is and how to install it (many many options). We can assume that Keptn users are technically savvy enough to be past that point.

That said, I do think pointing to an "as close to official small kubernetes distribution like kind" is useful.

Reply:
All I did here was provide links to the existing documentation about install and enable (the latter is Keptn specific, after all) and, just below, point to the instructions we have for setting up KinD.  If the user knows how to do these things, they don't need to click on those links.  Everyone agrees that we need to provide installation instructions in the Getting Started Guide, even though it is duplicating information that is elsewhere so requires double-maintenance.  And, if I'm new to Keptn, it's not totally obvious that I need a k8s cluster running Keptn even if all I want to do is run some tasks before and after my deployment that is not on k8s.  It seems to me that this just sets the stage, as it were.  We'll discuss ;-)